.test-infra/jenkins/LoadTestsBuilder.groovy

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+
+// Class for building Load Tests jobs and suites
+class LoadTestsBuilder {
+
+    private static Map<String, Object> defaultOptions = [
+            project             :'apache-beam-testing',
+            tempLocation        : 'gs://temp-storage-for-perf-tests/loadtests',
+    ]
+
+    enum Runner {
```

Comment:
This is common for Nexmark and load tests - can you move it to a common file and import here? Something like `CommonTestProperties.groovy` maybe?

Reply:
The same will apply to `TriggeringContext` once we use it in LoadTestBuilder

Reply:
Ok, I extracted common part for both suites. Thanks for suggestion.